cas-server-support-osf/src/main/java/org/scribe/builder/api/OrcidApi20.java

```diff
@@ -35,32 +35,19 @@
     private static final String AUTH_URL = "https://www.orcid.org/oauth/authorize";
 
     /** The token exchange url. */
-    private static final String TOKEN_URL = "https://%s.orcid.org/oauth/token";
-
-    /** The member flag. */
-    private final Boolean member;
+    private static final String TOKEN_URL = "https://orcid.org/oauth/token";
 
     /**
      * The default constructor.
      */
-    public OrcidApi20() {
-        this(Boolean.TRUE);
-    }
-
-    /**
-     * Create an instance of `OrcidApi20` with the `member` flag.
-     * @param member the member flag
-     */
-    public OrcidApi20(final Boolean member) {
-        this.member = member;
-    }
+    public OrcidApi20() {}
```

Comment:
As seen from line 41 - 56 in the deletions, `member` has been removed from the class. Now use the default constructor. I had thought about passing auth and token URLs here. However, that requires more changes on the ORCiD client side as well as `bean` init, which may break inheritance. So I left it for future improvement.